src/site/content/en/blog/optimize-vitals-lighthouse/index.md

```diff
@@ -0,0 +1,369 @@
+
+Optimizations in Chrome can remove the first render cycle for [optional fonts that are preloaded](/preload-optional-fonts) using `<link rel=preload>`. Rendering is blocked until the custom font has finished loading or a certain amount of time has passed, meaning no FOIT.
+
+Lighthouse highlights opportunities to preload fonts when one or more of them used `font-display: optional` and were not preloaded. 
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/GzqMzIzPVS1n5H11LdHf.png", alt="Lighthouse audit suggesting preloading optional fonts", width="800", height="545" %}
+</figure>
+
+Keep in mind that great care should be taken when considering preload for fonts as other options, such as [inlining font CSS at build-time](https://nextjs.org/blog/next-10-2#automatic-webfont-optimization), may also be able to get you far without as much risk of contending with requests for other important resources. [Preloading Fonts and the Puzzle of Prorities](https://andydavies.me/blog/2019/02/12/preloading-fonts-and-the-puzzle-of-priorities/) by Andy Davies is also a solid write-up on the trade-offs of font preloading.
+
+{% Aside %}It can seem counterintuitive to use `font-display: optional` on critical fonts. There is a [spec issue](https://github.com/w3c/csswg-drafts/issues/5924) open for discussing if a new `font-display` value should be added for such fonts. {% endAside %}
+
+### Avoid non-composited animations
+
+Animations which are non-composited can present themselves as janky on lower-end devices if heavy
+JavaScript tasks are keeping the main thread busy. Such animations can introduce layout shifts.
+
+If Chrome discovers an animation couldn't be composited, it reports it to a DevTools trace
+Lighthouse reads, allowing it to list which elements with animations weren't composited and for
+what reason. You can find these in the [Avoid non-composited
+animations](/non-composited-animations/) audit.
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/heGuYXKeMrUftMvfrDU7.png", alt="Audit for avoiding non-composited animations", width="800", height="528" %}
+</figure>
+
+### Debug First Input Delay / Total Blocking Time / Long Tasks
+
+First Input measures the time from when a user first interacts with a page (e.g. when they click a
+link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is
+actually able to begin processing event handlers in response to that interaction. Long JavaScript
+Tasks can impact this metric and the proxy for this metric, Total Blocking Time. 
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/LqBCtAXdByd4fBzoNc1K.png", alt="Audit for avoiding long main thread tasks", width="800", height="485" %}
+</figure>
+
+Lighthouse includes an [Avoid long main-thread tasks](/long-tasks-devtools/) audit which lists the
```

Comment:
worth calling out the 3p audit too that lists how much each party contributed to TBT?

Reply:
Added a reference in https://github.com/GoogleChrome/web.dev/pull/5237/commits/041d4dd7c61753ebcf4524d18a3e67e313b03a8a